proposed/security-disclosure.md

```diff
@@ -22,3 +22,99 @@ interpreted as described in [RFC 2119][].
+note that the version is no longer receiving security fixes.
+
+## Security Disclosure Process
+
+Every project MUST provide an email address in their security disclosure
+process description. If not specified otherwise, this email address is
+``security@[project domain]``. Projects SHALL NOT use contact forms.
+
+...?
+
+## Disclosure Discovery
+
+Every project MUST provide a link to its security vulnerability database in
+an obvious place. Ideally this should be on the main domain of the given
+project. This MAY be a sub-domain in case it is a sub-project of a larger
+initiative. If the project has a dedicated page for its disclosure process
+discovery then this is also considered a good place for this link.
+The link MAY use the custom link relation ``php-vuln-disclosures``,
+ie. for example
+``<link rel="php-vuln-disclosures" href="http://example.org/disclosures"/>``.
+
+Note that projects MAY choose to host their disclosure files on a domain
+other than their main project page. It is RECOMMENDED to not store the
+disclosures in a VCS as this can lead to the confusions about which branch
+is the relevant branch. If a VCS is used then additional steps should be taken
+to clearly document to users which branch contains all vulnerabilities for
+all versions. If necessary projects MAY however split vulnerability disclosure
+files by major version number. In this case
+
+## Disclosure Format
+
+The disclosure format is based on Atom [1], which in turn is based on XML. It
+leverages the "The Common Vulnerability Reporting Framework (CVRF) v1.1" [2].
+Specifically it leverages its dictionary [3] as its base terminology.
+
+The Atom extensions allow a structured description of the vulnerability to
+enable automated tools to determine if installed is likely affected by the
+vulnerability. However human readability is considered highly important and as
+such not the full CVRF is used.
+
+Note that for each vulnerability only a single entry should be created. In case
+any information changes the original file MUST be updated along with the last
+update field.
+
+Any disclosure uses ``entryType`` using the following tags from the Atom namespace:
+
+* title (short description of the vulnerability and affected versions)
+* summary (description of the vulnerability)
+* author (contact information)
+* published (initial publication date)
+* updated (date of the last update)
+* link (to reference more information)
+* id (project specific vulnerability id)
+
+In addition the following tags are added:
+
+* name (name of the product)
+* cve (unique CVE ID)
+* cwe (unique CWE ID)
+* severity (low, medium high)
+* affected (version(s) using composer syntax [4])
+* status (open, in progress, disputed, completed)
+* remediation (textual description for how to fix an affected system)
+* remediationType (workaround, mitigation, vendor fix, none available, will not fix)
+* remediationLink (URL to give additional information for remediation)
```

Comment:
a vulnerability might have multiple remediations, e.g.
* upgrade to the latest version
* delete a certain file (most of the times it is xmlrpc.php ;) ) if the above is not possible